src/cmd/uds.go

```diff
@@ -27,11 +27,20 @@ var createCmd = &cobra.Command{
 	Args:    cobra.MaximumNArgs(1),
 	Short:   lang.CmdBundleCreateShort,
 	PreRun: func(cmd *cobra.Command, args []string) {
-		if len(args) > 0 && !zarfUtils.IsDir(args[0]) {
-			message.Fatalf(nil, "(%q) is not a valid path to a directory", args[0])
+		pathToBundleFile := ""
+		if len(args) > 0 {
+			if !zarfUtils.IsDir(args[0]) {
+				message.Fatalf(nil, "(%q) is not a valid path to a directory", args[0])
+			}
+			pathToBundleFile = args[0] + "/"
```

Comment:
Instead of manually building out paths, we prefer to use `filepath.Join` in Go

Reply:
Updated